pkg/kubelet/cpumanager/policy.go

```diff
@@ -24,6 +24,7 @@ import (
 // Policy interface used by cpu manager, shall implement logic for cpu to pod
 // assignment
 type Policy interface {
+	IsUnderPressure() bool
```

Comment:
Could we move this to the state abstraction instead, and leave it out of the CPUManager and Policy interfaces?